packages/ui-extensions/src/docs/shared/components/Badge.ts

```diff
@@ -3,7 +3,7 @@ import {SharedReferenceEntityTemplateSchema} from '../docs-type';
 const data: SharedReferenceEntityTemplateSchema = {
   name: 'Badge',
   description:
-    "Use `s-badge` to inform merchants of the status of an object or of an action that's been taken.",
+    'Inform merchants about the status of an object or indicate that an action has been completed.',
```

Comment:
```suggestion
    'Inform users about the status of an object or indicate that an action has been completed.',
```

Since this is meant to be generic, lets replace "merchants" by "users".